### PR TITLE
feat: ability to show badges

### LINKED ISSUE #405: section to show cliparts
**Describe the feature you'd like**
creating a section to show all cliparts

**Would you like to work on the issue?**
YES

### PR DESCRIPTION
**Pull request by adityastic**
Fixes #405

Changes: 
- Added Option to show saved cliparts

Screenshots for the change:
In Drawer: 

Section:

### CODE REVIEW

File: app/src/main/res/values-zh-rCN/strings.xml [hunk 1]
```diff
@@ -89,4 +89,5 @@
     <string name="drawer_draw_badge">画剪贴画</string>
     <string name="clipart_saved_success">ClipArt已成功保存</string>
     <string name="clipart_saved_error">保存剪贴画时出错</string>
+    <string name="drawer_saved_cliparts">Saved Cliparts</string>
```
Comment (yashk2000) [inline app/src/main/res/values-zh-rCN/strings.xml]: Shouldn't it be in chinese here?

Comment (adityastic) [reply app/src/main/res/values-zh-rCN/strings.xml]: Yes, but I usually batch translate before releasing a version on playstore xD.

Comment (yashk2000) [reply app/src/main/res/values-zh-rCN/strings.xml]: Ok then :)

**Comment by adityastic** [comment]
@mariobehling @iamareebjamal Peer Review done. Please Review

File: app/src/main/res/values-zh-rCN/strings.xml (hunk 1 above)
Comment (iamareebjamal) [reply app/src/main/res/values-zh-rCN/strings.xml]: Change or remove

Comment (adityastic) [reply app/src/main/res/values-zh-rCN/strings.xml]: @iamareebjamal changing

**Comment by adityastic** [comment]
@iamareebjamal updated.

**Pull request merged**